Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -638,7 +639,6 @@ namespace Tensile
          double NumCUs, double TotalGranularity, int GlobalSplitU) const
     {
         StaticPerformanceModel spm;
-
         double beta = 0.0; // TODO-model - base on input value
```

Comment:
@amdkila @squiring - can we access the runtime beta value here?  Beta impacts the dynamic code paths in the kernel and runtime and thus the bandwidth that the kernel consumes.